server/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('../database/model.js');
+const seed = require(('../database/seed.js');)
+const helpers = require('../helpers/realtor.js');
+
+const PORT = 3000;
+
+app.use(bodyParser.json());
```

Comment:
just for future reference, but you can actually just do "app.use(express.json());". no need to install and require body parser 👍

Reply:
That's really good to know! Thank you!